mountpoint-s3/src/cli.rs

```diff
@@ -841,26 +880,34 @@ where
     tracing::trace!("using metadata TTL setting {metadata_cache_ttl:?}");
     filesystem_config.cache_config = CacheConfig::new(metadata_cache_ttl);
 
-    if let Some(path) = &args.cache {
-        let cache_limit = match args.max_cache_size {
-            // Fallback to no data cache.
-            Some(0) => None,
-            Some(max_size_in_mib) => Some(CacheLimit::TotalSize {
-                max_size: (max_size_in_mib * 1024 * 1024) as usize,
-            }),
-            None => Some(CacheLimit::default()),
-        };
-        if let Some(cache_limit) = cache_limit {
-            let cache_config = DiskDataCacheConfig {
-                block_size: args.cache_block_size_in_bytes(),
-                limit: cache_limit,
-            };
-            let cache_key = env_unstable_cache_key();
-            let managed_cache_dir = ManagedCacheDir::new_from_parent_with_cache_key(path, cache_key)
-                .context("failed to create cache directory")?;
+    match (args.disk_data_cache_config(), args.cache_express_bucket_name()) {
+        (None, Some(express_bucket_name)) => {
+            tracing::debug!("using express cache");
```

Comment:
Either drop to `trace` level, or review the user messages (here and in the other cases) to match the args description/docs. Things to consider:
"cache" -> "object content cache" (vs metadata)
"express" -> "S3 Express One Zone"
"multilevel" -> not sure we have settled on a user-facing name

Reply:
reduced to `trace` level, tried to make messages more descriptive